### DISCUSSION TITLE
[Feature Requests] great swype features

### DISCUSSION THREAD

**Discussion by adeyo50** [florisboard/florisboard#1282] [Feedback]
If anyone has ever used the original swype keyboard or swiftkey there are a few "speedy" features I feel I can no longer live without. Here they are...

1 while tapping the delete key backspaces and deletes a stroke, long pressing deletes an entire word then continuing to hold starts quickly jumping back (deleting) additional words until the key is released

2 for speedy use of the often used secondary keys (for ex ? ! , Etc) ...instead of long pressing, I simply swipe from the key to the spacebar. It's amazing how much faster this can be than long pressing

**Comment by anubis-genix** [comment]
Glad to see another Swype user. I already opened an "issue/feature request" for your second suggestion.
It's one of the best features in Swype. Once you're used to it, you can't live without it :D
https://github.com/florisboard/florisboard/issues/814

**Comment by adeyo50** [comment]
Glad to hear that! I'm not a techy and, before now, have never posted on
github before. I'm watching the development of the AnySoftKeyboard,
Florisboard and even openboard (wondering if they will add Swype or gesture
typing) in the hopes that somebody with privacy in mind will rival Swype or
Swiftkey some day.

Regarding Florisboard, do have any clue when they may try to implement
thowe Swype features... if at all, that is?

Thanks,

Adam
